api/DOMParser.json

```diff
@@ -223,10 +223,10 @@
                 "version_added": true
               },
               "safari": {
-                "version_added": "3.2"
+                "version_added": "3"
```

Comment:
This is the SVG support, did you track down a commit or where is the version from?

Reply:
This one's admittedly a guess, I figure that `DOMParser` support would have been around for SVGs when SVG support was first introduced in Safari.  I've just checked the code, however, and it seems to suggest what I assumed.

https://trac.webkit.org/browser/webkit/tags/old/Safari-5522.11/WebCore/dom/DOMImplementation.cpp#L227
https://trac.webkit.org/browser/webkit/tags/old/Safari-5522.11/WebCore/xml/DOMParser.cpp#L33

Reply:
That looks like the right code for this, thanks for verifying!